benchmarks/results/producer.md

```diff
@@ -0,0 +1,17 @@
+## Producer benchmarks
+
+This section includes benchmarks for single and sink producers.
+
+## RC7
+
+### 10iterations 1fork 1thread
```

Comment:
Shouldn't we compare results of the same fork / threads setup?

Reply:
Yep they should, will address that, although I think I just added them more as a base for future api changes since the producer logic was untouched.